Source/Tools/FEXLoader/LinuxSyscalls/GdbServer.cpp

```diff
@@ -441,7 +494,7 @@ fextl::string buildTargetXML() {
 
       // GPRs
       for (uint32_t i = 0; i < FEXCore::Core::CPUState::NUM_GPRS; i++) {
-        reg(FEXCore::Core::GetGRegName(i), "int64", 64);
+        reg(GetGRegName(i), "int64", 64);
```

Comment:
Does GDB not expect a specific ordering here? Are we consistent with what other gdbserver implementations return?

Reply:
gdb doesn't expect a specific ordering but it will end up displaying the order we provide it when you do a `info regs all`

Reply:
Fair enough, no need to worry about it here then.